cardano-crypto-praos/src/Cardano/Crypto/VRF/PraosBatchCompat.hs

```diff
@@ -161,47 +161,53 @@ newtype Output = Output { unOutput :: ForeignPtr OutputValue }
 
 -- Raw low-level FFI bindings.
 --
-foreign import ccall "crypto_vrf_ietfdraft10_proofbytes_batchcompat" crypto_vrf_ietfdraft10_proofbytes_batchcompat :: CSize
-foreign import ccall "crypto_vrf_ietfdraft10_publickeybytes" crypto_vrf_ietfdraft10_publickeybytes :: CSize
-foreign import ccall "crypto_vrf_ietfdraft10_secretkeybytes" crypto_vrf_ietfdraft10_secretkeybytes :: CSize
-foreign import ccall "crypto_vrf_ietfdraft10_seedbytes" crypto_vrf_ietfdraft10_seedbytes :: CSize
-foreign import ccall "crypto_vrf_ietfdraft10_outputbytes" crypto_vrf_ietfdraft10_outputbytes :: CSize
-
-foreign import ccall "crypto_vrf_ietfdraft10_publickeybytes" io_crypto_vrf_ietfdraft10_publickeybytes :: IO CSize
-foreign import ccall "crypto_vrf_ietfdraft10_secretkeybytes" io_crypto_vrf_ietfdraft10_secretkeybytes :: IO CSize
-
-foreign import ccall "crypto_vrf_ietfdraft10_keypair_from_seed" crypto_vrf_ietfdraft10_keypair_from_seed :: VerKeyPtr -> SignKeyPtr -> SeedPtr -> IO CInt
-foreign import ccall "crypto_vrf_ietfdraft10_sk_to_pk" crypto_vrf_ietfdraft10_sk_to_pk :: VerKeyPtr -> SignKeyPtr -> IO CInt
-foreign import ccall "crypto_vrf_ietfdraft10_sk_to_seed" crypto_vrf_ietfdraft10_sk_to_seed :: SeedPtr -> SignKeyPtr -> IO CInt
-foreign import ccall "crypto_vrf_ietfdraft10_prove_batchcompat" crypto_vrf_ietfdraft10_prove_batchcompat :: ProofPtr -> SignKeyPtr -> Ptr CChar -> CULLong -> IO CInt
-foreign import ccall "crypto_vrf_ietfdraft10_verify_batchcompat" crypto_vrf_ietfdraft10_verify_batchcompat :: OutputPtr -> VerKeyPtr -> ProofPtr -> Ptr CChar -> CULLong -> IO CInt
-
-foreign import ccall "crypto_vrf_ietfdraft10_proof_to_hash_batchcompat" crypto_vrf_ietfdraft10_proof_to_hash_batchcompat :: OutputPtr -> ProofPtr -> IO CInt
+foreign import ccall "crypto_vrf_ietfdraft13_bytes_batchcompat" crypto_vrf_ietfdraft13_proofbytes_batchcompat :: CSize
```

Comment:
I think it would be good to keep them named the same for consistency sake with all foreign functions:
```suggestion
foreign import ccall "crypto_vrf_ietfdraft13_bytes_batchcompat" crypto_vrf_ietfdraft13_bytes_batchcompat :: CSize
```